lib/ssl/src/tls_client_connection_1_3.erl

```diff
@@ -486,6 +486,8 @@ wait_finished(Type, Msg, State) ->
                  term(), #state{}) ->
           gen_statem:state_function_result().
 %%--------------------------------------------------------------------
+connection(info, Msg, State) ->
+    tls_gen_connection:handle_info(Msg, connection, State);
```

Comment:
So I am not following the logic here!  By removing
this clause we will get tls_gen_connection_1_3:connection -> ssl_gen_statem:connection -> tls_gen_connection:handle_info

Reply:
I don't understand what you are asking?

Reply:
I have removed two unnecessary function calls.